test-interface/src/main/scala/org/scalajs/testinterface/internal/FrameworkLoader.scala

```diff
@@ -7,8 +7,10 @@ import sbt.testing.Framework
 private[internal] object FrameworkLoader {
 
   def loadFramework(frameworkName: String): Framework = {
+    val exportsNamespace =
+      scala.scalajs.runtime.environmentInfo.exportsNamespace
```

Comment:
So just to double check: This has been around since 0.6.0, right? So binary compatibility is in any case not an issue.